site/content/en/docs/Getting Started/first_match.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Create Your First Match"
 linkTitle: "First Match"
-weight: 1
+weight: 2
```

Comment:
I want to keep first match before the concepts.  I think it's far easier to carve out a small example to give a point of reference before throwing all of the concepts at the reader which they have to construct to a working mental model.

Reply:
To clarify - First Match today has a section on concepts - and a suggestion is splitting that out into a separate page - before first match. What is confusing today is - there is a first match - and that has a first match concepts - and then the next page has a core concepts. Here is an excerpt from user feedback:

Seems a little odd that we have two sections (“FIrst Match” does the same) that show “core concepts” - are they all core? Why am I only learning about these parts now?

I agree with this one. I am removing the OM core concepts (as the page exists today) from Getting Started completely and moving them to a Building a Matchmaker Guide - where you expose methodically all the concepts for all the components needed.

The concepts.md can 
- either go away completely - go getting started has just first_match
- can preceed first_match to explain the concepts needed for first match(that are in the fm page today) 

Your call. I ll remove that page for now.